src/Core/Context/CurrentContext.cs

```diff
@@ -97,6 +98,11 @@ public async virtual Task BuildAsync(HttpContext httpContext, GlobalSettings glo
         {
             ClientVersion = cVersion;
         }
+
+        if (httpContext.Request.Headers.TryGetValue("Is-Prerelease", out var clientVersionIsPrerelease))
+        {
+            ClientVersionIsPrerelease = clientVersionIsPrerelease == "true";
```

Comment:
@withinfocus are we OK with this being `true` and not an integer value?  I can change the header if that would make this less brittle?

Reply:
They're all strings in the end, but if you made this 1 or 0 I think it aligns _slightly_ better with some others out there. Lemme know.

Reply:
Adjusted this with https://github.com/bitwarden/clients/pull/11909.